pyterrier_rag/readers/_base.py

```diff
@@ -52,6 +52,13 @@ def __post_init__(self):
             else:
                 self.backend = self.backend.text_generator()
 
+    def transform_inputs(self):
```

Comment:
I don't think this one needs to rely on the escape hatches, since it's a DataFrame `transform`

Reply:
the transform calls the prompt transformer, which is not dataframe based...